tutorials/chapter1.md

```diff
@@ -416,6 +456,13 @@ As it can be seen in this code, the ```kpsr::high_perf::DataMultiplexer``` has a
 	* Listener will have access to the latest data only. If a listener is slow in processing, data multiplexer will skip old data and only provide it with the latest one.
 * There is no start or stop methods. The start is implicitly triggered when a listener is registered in the ```Subscriber```.
 
+* Then, there are five threads running:
+	* The main thread
+	* Two DataMultiplexer listeners threads (due to listeners invocations are independent in data multiplexer)
```

Comment:
> due to listeners invocations are independent in data multiplexer

as each listener is started in its own thread

Reply:
Done